test/run/nat-shift.mo

```diff
@@ -0,0 +1,37 @@
+import { debugPrint; shiftLeft; shiftRight } = "mo:⛔"
```

Comment:
Instead of producing lots of output and relying on diff and visual inspection. why not just compare the result against the naive implementation using multiplication/division by recursively computed power of 2?

Reply:
Yeah, good idea. I started out wanting to see the results, but asserting makes a lot of sense now.

Reply:
Done.